pcsx2/USB/usb-eyetoy/cam-linux.cpp

```diff
@@ -7,6 +7,7 @@
 #include "jo_mpeg.h"
 #include "common/Console.h"
 
+#include <mutex>
```

Comment:
lock_guard is defined in the same header as the mutex we're using it on, so we shouldn't need to add an include for it